pom.xml

```diff
@@ -46,25 +50,14 @@
     </modules>
 
     <properties>
-        <jersey.version>2.19</jersey.version>
-        <confluent-common.version>3.3.0-SNAPSHOT</confluent-common.version>
-        <kafka.version>0.11.0.0-SNAPSHOT</kafka.version>
-        <kafka.scala.version>2.11</kafka.scala.version>
-        <log4j.version>1.7.6</log4j.version>
-        <restutils.version>3.3.0-SNAPSHOT</restutils.version>
+        <apache.directory.server.version>2.0.0-M22</apache.directory.server.version>
+        <apache.directory.api.version>1.0.0-M33</apache.directory.api.version>
         <avro.version>1.7.7</avro.version>
+        <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
+        <exec-maven-plugin.version>1.2.1</exec-maven-plugin.version>
         <jackson.version>2.5.4</jackson.version>
-        <junit.version>4.12</junit.version>
-        <easymock.version>3.3.1</easymock.version>
-        <powermock.version>1.6.2</powermock.version>
-        <powermock.api.easymock.version>1.6.2</powermock.api.easymock.version>
+        <log4j.version>1.7.6</log4j.version>
```

Comment:
I think this is from before, but I think this is actually the slf4j version and already set as `slf4j.version` in one of the parent poms, so we might want to get consistent while we are at it. (Also, it was at a 1.7.21 instead of 1.7.6